src/Symfony/Component/Workflow/SupportStrategy/InstanceOfSupportStrategy.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\SupportStrategy;
+
+use Symfony\Component\Workflow\WorkflowInterface;
+
+/**
+ * @author Andreas Kleemann <akleemann@inviqa.com>
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ */
+final class InstanceOfSupportStrategy implements WorkflowSupportStrategyInterface
+{
+    private $className;
+
+    public function __construct(string $className)
+    {
+        $this->className = $className;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supports(WorkflowInterface $workflow, $subject): bool
+    {
+        return $subject instanceof $this->className;
+    }
+
+    /**
+     * @return string
```

Comment:
aren't we trying to get rid of these kind docblocks?

Reply:
👍  useless